system/lib/libc/musl/src/conf/sysconf.c

```diff
@@ -88,9 +88,9 @@ long sysconf(int name)
 		[_SC_THREAD_THREADS_MAX] = -1,
 		[_SC_THREAD_ATTR_STACKADDR] = VER,
 		[_SC_THREAD_ATTR_STACKSIZE] = VER,
-		[_SC_THREAD_PRIORITY_SCHEDULING] = VER,
-		[_SC_THREAD_PRIO_INHERIT] = 200809, // XXX EMSCRIPTEN replace -1,
-		[_SC_THREAD_PRIO_PROTECT] = 200809, // XXX EMSCRIPTEN replace -1,
```

Comment:
Any idea why this was done?  @juj do you remember why this was needed?

Reply:
I changed these back to `-1` (i.e. unsupported) as we don't actually support priority inherit protection or the priority ceiling attribute in mutexes.

As an aside, could it perhaps be that this sysconf mimicked a native linux (non-musl) environment? I also noticed that these are increased:
https://github.com/emscripten-core/emscripten/blob/097cfb8726001cf81343f1a9f61e06b5074ace85/system/lib/libc/musl/src/conf/sysconf.c#L86-L87
https://github.com/emscripten-core/emscripten/blob/097cfb8726001cf81343f1a9f61e06b5074ace85/system/include/libc/limits.h#L63-L64

(I'm not sure what the best approach is, perhaps sysconf should _not_ correspond to the actual wasm/musl limits)